# Peut-on prédire une attaque cardiaque ?

In [14]:
import pandas as pd
import numpy as np
import seaborn as sns

heart_data = pd.read_csv('C:/Users/maxime.avrillonthade/OneDrive - VINCI Energies/Documents/Cours H3 Hitema/analyse de donnees/Projet/heart.csv')
Saturation_data = pd.read_csv('C:/Users/maxime.avrillonthade/OneDrive - VINCI Energies/Documents/Cours H3 Hitema/analyse de donnees/Projet/o2Saturation.csv')

In [15]:
heart_data.head()

age  sex  cp  trtbps  chol  fbs  restecg  thalachh  exng  oldpeak  slp  \
0   63    1   3     145   233    1        0       150     0      2.3    0   
1   37    1   2     130   250    0        1       187     0      3.5    0   
2   41    0   1     130   204    0        0       172     0      1.4    2   
3   56    1   1     120   236    0        1       178     0      0.8    2   
4   57    0   0     120   354    0        1       163     1      0.6    2   

   caa  thall  output  
0    0      1       1  
1    0      2       1  
2    0      2       1  
3    0      2       1  
4    0      2       1

📄A propos de l'ensemble de données
Cet ensemble de données contient 14 colonnes :

Age : Âge du patient

Sex : sexe du patient

cp : Type de douleur thoracique type de douleur thoracique

Valeur 1 : angine typique

Valeur 2 : angine atypique

Valeur 3 : douleur non angineuse

Valeur 4 : asymptomatique

trtbps : pression artérielle au repos (en mm Hg)

chol : cholestérol en mg/dl récupéré via le capteur BMI

fbs : (glycémie à jeun > 120 mg/dl)

1 = vrai

1 = vrai 0 = faux

rest_ecg : résultats de l'électrocardiographie au repos

Valeur 0 : normal

Valeur 1 : présentant une anomalie de l'onde ST-T (inversions de l'onde T et/ou élévation ou dépression de l'onde ST de > 0,05 mV)

Valeur 2 : présentant une hypertrophie ventriculaire gauche probable ou certaine selon les critères d'Estes.

thalach : fréquence cardiaque maximale atteinte

exang : angine de poitrine induite par l'exercice

1 = oui

0 = non


ca : nombre de vaisseaux principaux (0-3)

cible :

0= moins de risque de crise cardiaque

1= plus de risque d'infarctus